packages/flutter_test/lib/src/binding.dart

```diff
@@ -1258,6 +1275,12 @@ enum LiveTestWidgetsFlutterBindingFramePolicy {
   /// directly to run (either by using [WidgetTester.pumpBenchmark] or invoking
   /// [Window.onBeginFrame] and [Window.onDrawFrame]).
   ///
+  /// This allows all frame requests from the engine to be serviced, and allows
+  /// all frame requests that are artificially triggered to be serviced, but
```

Comment:
Nit: `triggered` -> `[pump]ed`?

Reply:
I'm not sure I should use `pump` here because it's not only from pump but also from animation. I'll leave this to @sfshaza2 for language advice.

Reply:
Perhaps:
```
This allows all frame requests from the engine to be serviced (including frame
requests that are artificially triggered), but...
```

I wouldn't use pumped instead of triggered.

Reply:
I'll keep the original version because `artificially triggered` is from test file, not the engine.